gebrueder-weiss-woocommerce.php

```diff
@@ -23,20 +23,29 @@
 require __DIR__ . '/vendor/autoload.php';
 
 use GbWeiss\includes\GbWeiss;
+use GbWeiss\includes\OAuth\OAuthAuthenticator;
 use GbWeiss\includes\OrderStateRepository;
 
+/**
+ * Use Dotenv to set required environment variables and load .env file in root
+ */
+$dotenv = \Dotenv\Dotenv::createImmutable(__DIR__);
+$dotenv->safeLoad();
+
 add_action("init", function () {
     if (!GbWeiss::checkPluginCompatabilityAndPrintErrorMessages()) {
         return;
     };
 
     $plugin = GbWeiss::getInstance();
+    $authenticationClient = new OAuthAuthenticator(new GuzzleHttp\Client());
+    $authenticationClient->setAuthenticationEndpoint($_ENV['GEBRUEDER_WEISS_OAUTH_URL'] ?: "https://apitest.gw-world.com:443/authorize");
```

Comment:
We have to check here if the environmentvariable exists. Otherwise PHP will fail with an undefined index.
```suggestion
    $authEndpoint = array_key_exists('GEBRUEDER_WEISS_OAUTH_URL', $_ENV) ? $_ENV['GEBRUEDER_WEISS_OAUTH_URL'] : "https://apitest.gw-world.com:443/authorize"
    $authenticationClient->setAuthenticationEndpoint($authEndpoint);
```